source/_docs/relaunch.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Relaunch Existing Pantheon Site
+description: Take a new site live by moving custom domains from one Site Dashboard to another, with minimal HTTPS interruptions.
+tags: [dns]
+categories: []
+---
+Sites are considered launched on Pantheon once traffic is routed through custom domain(s). Relaunching a previously launched site is done by rerouting traffic from the existing Site Dashboard to an entirely new Site Dashboard.
+
+<div class="alert alert-info">
+<h4 class="info">Note</h4>
+<p markdown="1">The relaunch process applies exclusively to live sites already hosted on Pantheon. Otherwise, refer to [Launch Essentials](/docs/guides/launch/).</p></div>
+
+## Before You Begin
+- Log in to the new Site Dashboard on Pantheon
+- Open a second tab for the old Site Dashboard on Pantheon
+- In a third tab, log in to the domain's DNS service provider (e.g., Cloudflare, Amazon Route 53, etc.)
+- Examine existing records pointing to Pantheon
+  {% include("content/standard-dns-config.html") %}
+- Lower the TTL of existing DNS records to minimize impact of upcoming DNS changes
+  {% include("ttl.twig") %}
+
+## Roles & Permissions
+The permission to manage billing and plans is granted only to the role of **Site Owner** / **Organization Administrators**. Other roles do not have access as described on this page.
+
+<div class="alert alert-info">
+<h4 class="info">Note</h4>
+<p markdown="1">If you need to assume site and billing ownership, the current Site Owner must [transfer it to you directly](/docs/site-billing#transfer-ownership-and-billing-for-this-site).</p></div>
+
+## Relaunch Procedure
+1. In the new Site Dashboard, [upgrade the site from free to a paid plan](/docs/site-plan/#change-site-plan).
```

Comment:
I think we might want to do one domain at a time, quickly remove one from old, quickly add domain to new (even recommend copy/paste to do this quickly?). If they have many domains they'll especially want to do it one at a time to ensure the domain  is always routing to a Pantheon site (so you don't get the 404 no Pantheon site error)